pkg/mls/store/queries.sql

```diff
@@ -131,49 +139,99 @@ INSERT INTO welcome_messages(installation_key, data, installation_key_data_hash,
 RETURNING
 	*;
 
--- name: QueryGroupMessagesAsc :many
+-- name: GetAllGroupMessages :many
+SELECT
+	*
+FROM
+	group_messages
+ORDER BY
+	id ASC;
+
+-- name: QueryGroupMessages :many
 SELECT
 	*
 FROM
 	group_messages
 WHERE
 	group_id = @group_id
+ORDER BY
```

Comment:
I'm using this fancy case statement for sorting for now. It seems to work fine, although I'm a little worried about perf. Sometimes using a case statement will prevent the query planner from using indexes. Will try and test

Reply:
Curious about how you would do this. Would `pgbench` and tools that run thorough SQL transcripts be considered a practical approach?